src/pages/iou/WaypointEditor.tsx

```diff
@@ -141,10 +123,10 @@ function WaypointEditor({route: {params: {iouType = '', transactionID = '', wayp
         // Therefore, we're going to save the waypoint as just the address, and the lat/long will be filled in on the backend
         if (isOffline && waypointValue) {
             const waypoint = {
-                lat: null,
-                lng: null,
+                lat: -1,
+                lng: -1,
                 address: waypointValue,
-                name: null,
+                name: undefined,
```

Comment:
Why can't we keep the old logic?

Reply:
`saveWaypoint` function parameter type is `OnyxTypes.RecentWaypoint` where lng and lat is required thats why its added , do you think  I should adjust a type?

Reply:
I think yes, if they have possibility of being `null`/`undefined`. You can mark them as optional and here change to `undefined`.